### PR TITLE
Kdp updates

### PR DESCRIPTION
**Pull request by bbrewer-forge**

### CODE REVIEW

File: kubernetes-data-platform/templates/_helpers.tpl [hunk 1]
```diff
@@ -118,7 +118,6 @@ Create a common label block
 */}}
 {{- define "kdp.labels" -}}
 environment: {{ .Values.global.environment }}
-chart: {{ .Chart.Name }}-{{ .Chart.Version }}
```
Comment (bbrewer-forge) [inline kubernetes-data-platform/templates/_helpers.tpl:121]: This is the cause of the immutable field errors we get when upgrading this chart

File: kubernetes-data-platform/templates/metastore/metastore-site.yaml [hunk 2]
```diff
@@ -28,7 +28,9 @@ data:
       </property>
       <property>
         <name>javax.jdo.option.ConnectionURL</name>
-        <value>jdbc:postgresql://{{ template "kdp.postgres.fullname" . }}/{{ .Values.postgres.db.name }}</value>
+        {{- $fullname := include "kdp.postgres.fullname" . -}}
+        {{- $hostname := default $fullname .Values.postgres.service.externalAddress -}}
```
Comment (bbrewer-forge) [inline kubernetes-data-platform/templates/metastore/metastore-site.yaml:32]: helm's default is backwards FYI

File: kubernetes-data-platform/templates/_helpers.tpl (hunk 1 above)
Comment (LtChae) [reply kubernetes-data-platform/templates/_helpers.tpl:121]: Glad you found this. Mighty confusing this morning.

Comment (bbrewer-forge) [reply kubernetes-data-platform/templates/_helpers.tpl:121]: It will require one last chart delete, but that "should" be it

**Pull request merged**